helix-core/src/test/java/org/apache/helix/integration/controller/TestControllerLeadershipChange.java

```diff
@@ -117,16 +258,17 @@ public void testMissingTopStateDurationMonitoring() throws Exception {
     // Resource lost top state, and manager1 lost leadership for 2000ms, because manager1 will
     // clean monitoring cache after re-gaining leadership, so max value of hand off duration should
     // not have such a large value
-    Assert.assertTrue((long) beanServer.getAttribute(resourceMBeanObjectName,
-        "PartitionTopStateHandoffDurationGauge.Max") < 500);
+    Assert.assertTrue((long) beanServer
+        .getAttribute(resourceMBeanObjectName, "PartitionTopStateHandoffDurationGauge.Max") < 500);
 
     participant.syncStop();
     manager1.disconnect();
     manager2.disconnect();
     deleteCluster(clusterName);
   }
 
-  private void setLeader(HelixManager manager) throws Exception {
+  private void setLeader(HelixManager manager)
```

Comment:
nit, I believe the latest style file has changed exception throws to non-wrapping. Please update your local style configuration.

Reply:
good catch. updated